test/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.t.sol

```diff
@@ -1160,7 +1160,7 @@ contract FM_BC_Bancor_Redeeming_VirtualSupplyV1Test is ModuleTest {
     /* Test getVirtualIssuanceSupply() */
 
     function testGetVirtualIssuanceSupply(uint _supply) public {
-        vm.assume(_supply > 0);
+        vm.assume(_supply > 10);
```

Comment:
Should we use `INITIAL_ISSUANCE_SUPPLY` here just to be clear of where this number comes from?

Reply:
Not doing this, not needed.